docs/process.rst

```diff
@@ -1301,6 +1304,9 @@ For example::
       """
     }
 
+.. warning:: When combined with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 
```

Comment:
```suggestion
.. note:: When combined with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 
```

Better to use `note` instead of a `warning`